pkg/operation/seed/seed.go

```diff
@@ -760,30 +760,19 @@ func BootstrapCluster(ctx context.Context, k8sGardenClient, k8sSeedClient kubern
 	}
 
 	// Deploy component specific resources
-	var bootstrapFunctions []flow.TaskFn
-	for _, componentFn := range []component.BootstrapSeed{
-		clusterautoscaler.BootstrapSeed,
-		func(ctx context.Context, c client.Client, namespace, version string) error {
-			image, err := imageVector.FindImage(
-				common.EtcdDruidImageName,
-				imagevector.RuntimeVersion(k8sSeedClient.Version()),
-				imagevector.TargetVersion(k8sSeedClient.Version()),
-			)
-			if err != nil {
-				return err
-			}
-
-			var imageVectorOverwrite *string
-			if val, ok := imageVectorOverwrites[etcd.Druid]; ok {
-				imageVectorOverwrite = &val
-			}
+	bootstrapComponents, err := bootstrapComponents(k8sSeedClient, v1beta1constants.GardenNamespace, imageVector, imageVectorOverwrites)
+	if err != nil {
+		return err
+	}
 
-			return etcd.BootstrapSeed(ctx, c, namespace, version, image.String(), imageVectorOverwrite)
-		},
-	} {
+	var bootstrapFunctions []flow.TaskFn
+	for _, componentFn := range bootstrapComponents {
 		fn := componentFn
 		bootstrapFunctions = append(bootstrapFunctions, func(ctx context.Context) error {
-			return fn(ctx, k8sSeedClient.Client(), v1beta1constants.GardenNamespace, k8sSeedClient.Version())
+			if err := fn.Deploy(ctx); err != nil {
+				return err
+			}
+			return fn.Wait(ctx)
```

Comment:
Alternatively, you can use:
```suggestion
			return component.OpWaiter(componentFn).Deploy(ctx)
```